legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -0,0 +1,177 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This provides the service for involuntary dissolution."""
+from sqlalchemy import and_, exists, func, not_, or_, text
+from sqlalchemy.orm import aliased
+
+from legal_api.models import Batch, BatchProcessing, Business, Filing, db
+
+
+DEFAULT_MIN_DATE = func.date('1800-01-01 00:00:00+00:00')
+
+
+class InvoluntaryDissolutionService():
+    """Provides services to get information for involuntary dissolution."""
+
+    @staticmethod
+    def get_businesses_eligible_count():
+        """Return the number of businesses eligible for involuntary dissolution."""
```

Comment:
Do we have a check for a business in Limited Restoration yet?

![image](https://github.com/bcgov/lear/assets/6685223/8f30bb2f-2b12-4602-9138-18c3664a0c39)

Reply:
Not yet, will add it.

Reply:
Updated.